docs/concepts/linq.rst

```diff
@@ -0,0 +1,268 @@
+LINQ
+====
+
+What is it?
+-----------
+
+LINQ provides language-level querying capabilities and `higher-order function <https://en.wikipedia.org/wiki/Higher-order_function>`_ API to C# and VB (note that F# already has this capability by nature of being a functional programming language).  It is used either via native query syntax,
+
+.. code-block:: c#
+
+	var dotNetProgrammers = from p in programmers
+	                        where p.IsNewToLINQ
+	                        select new LINQExpert(p);
+	
+Or as a higher-order function via the ``IEnumerable<T>`` interface,
+
+.. code-block:: c#
+
+	var dotNetProgrammers = programmers.Where(p => IsNewToLINQ)
+	                                   .Select(p => new LINQExpert(p));
+									   
+as a way to write expressive, declarative code.
```

Comment:
I find this form a bit confusing, where a sentence is broken by an example. I've seen it before, but as a reader, I prefer the sentences before and after examples to be separate and complete.